app/controllers/tmdb_controller.rb

```diff
@@ -135,12 +133,11 @@ def director_search
   end
 
   def discover_search
-    form_params = %i[sort_by date genre actor actor2 company mpaa_rating year_select page]
+    form_params = %i[sort_by date year genre actor_name company mpaa_rating timeframe page]
```

Comment:
I removed the 2 actor params and replaced it with a single actor name. The advanced search form does not support more than one actor. Those params had been in place back when we were relying on the `discover_search` method to do multiple things (now handled in `get_common_movies_between_multiple_actors`). Now that we've broken it out into different purposes, this method is only accepting the params it requires.